code/game/objects/items.dm

```diff
@@ -186,7 +186,10 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	. = ..()
 
 	. += "[gender == PLURAL ? "They are" : "It is"] a [weightclass2text(w_class)] item."
-
+	
+	if(HAS_TRAIT(src, TRAIT_NO_STORAGE))
+		. += "[gender == PLURAL ? "They are" : "It is"] too bulky, fragile, or cumbersome to fit in a container."
```

Comment:
I am VERY open to suggestions here. Something that clearly and unambiguously tells the player "this thing does not go in backpacks" without breaking being meta/immersion breaking.

Reply:
"impossible to put in a container"